tutorials/ui/gui_skinning.rst

```diff
@@ -1,183 +1,240 @@
-resource. Theme contains all the information required to change the
-entire visual styling of all controls. 
-
-A Theme can be applied to any control node in the scene. As a result,
-all children and grand-children controls will use that same theme, too
-(unless another theme is specified further down the tree). If a value is
-not found in a theme, it will be searched in themes higher up in the
-hierarchy, towards the root. If nothing was found, the default theme is
-used. This system allows for flexible overriding of themes in complex
-user interfaces.
-
-.. attention::
-   
-   Don't use the custom theme option in the Project Settings, as there
-   are known bugs with theme propagation. Instead, apply your theme to the
-   root Control node's Theme property. It will propagate to instanced scenes
-   automatically. To get correct theming in the editor for instanced scenes,
-   you can apply the theme resource to the instanced scene's root node as well.
-
-Creating a theme
-----------------
-
-Themes can be created from any control node. Select a control node in the scene
-hierarchy, then in the inspector go to the theme property. From there you can
-select **New Theme**.
-
-.. image:: img/new_theme.png
+Introduction to GUI skinning
+============================
 
-This will create an empty theme and open up the theme editor.
+It is essential for a game to provide clear, informative and yet visually
+pleasing user interface to its players. While :ref:`Control <class_Control>`
+nodes come with a decently functional look out of the box, there is always
+a room for uniqueness and case-specific tuning. For this purpose Godot engine
+includes a system for GUI skinning (or theming), which allows you to customize
+the look of every control in your user interface, including your custom controls.
 
-.. image:: img/theme_editor.png
+Here is an example of this system in action, which is radically different from
+the default UI theme of the engine:
 
-In the theme editor you can customize everything about a theme except for
-the default font the theme uses. That can only be customized in the inspector under
-the selected theme.
-
-.. image:: img/default_font.png
-
-Theme items
------------
+.. figure:: img/tank-kings-by-winterpixel-games.png
+   :align: center
 
-In the theme editor, next to the default preview window, is where you make changes
-to your theme. Clicking the plus button opens the **Add item Type** menu.
+   A "Gear Up!" screen in Tank Kings, courtesy of Winterpixel Games
 
-.. image:: img/add_item_type.png
+Beyond achieving a unique look for your game this system also enables developers
+to provide customization options to the end users, including accessibility settings.
+UI themes are applied in a cascading manner (i.e. they propagate from parent
+controls to their children), which means that font settings or adjustments
+for colorblind can be applied in a single place and affect the entire UI tree.
```

Comment:
```suggestion
for color blindness can be applied in a single place and affect the entire UI tree.
```
or colorblind users?

Reply:
Probably the second one, "adjustments for color blindness" sounds wrong.

Reply:
Probably the german in me then, we like our bureaucratic sentences with lots of proper nouns :P Go ahead